requirements-test.txt

```
@@ -3,3 +3,4 @@ coveralls
 flake8>=2.1.0
 tox>=2.0
 django-app-helper
+https://github.com/Aiky30/djangocms-versioning/tarball/feature/django-32-compat#egg=djangocms-versioning
```

```suggestion
https://github.com/django-cms/djangocms-versioning/tarball/master#egg=djangocms-versioning
```

This was an old pin before 3.2 changes were merged.